app/views/common/_pagination.html.erb

```diff
@@ -1,5 +1,5 @@
 <form
-  class='content-view-pf-pagination table-view-pf-pagination paginate' id='pagination'
+  class='col-md-12 <%= "audits-pagination" if @page_header == "Audits" %> content-view-pf-pagination table-view-pf-pagination paginate' id='pagination'
```

Comment:
The page header might change based on localization. Could we override the color in css using dome css selector rather than doing if here?